src/schema/objects/columns.yaml

```diff
@@ -12,23 +12,7 @@ abbreviation:
   type: string
 acq_time:
   name: acq_time
-  description: |
-    Acquisition time refers to when the first data point in each run was acquired.
-    Furthermore, if this header is provided, the acquisition times of all files that
-    belong to a recording MUST be identical.
-
-    Datetime should be expressed as described in [Units](/02-common-principles.html#units).
-
-    For anonymization purposes all dates within one subject should be shifted by a randomly
-    chosen (but consistent across all recordings) number of days.
-    This way relative timing would be preserved, but chances of identifying a person based
-    on the date and time of their scan would be decreased.
-    Dates that are shifted for anonymization purposes SHOULD be set to the year 1925 or earlier
-    to clearly distinguish them from unmodified data.
-    Note that some data formats do not support arbitrary recording dates.
-    For example, the [EDF](https://www.edfplus.info/)
-    data format can only contain recording dates after 1985.
-    Shifting dates is RECOMMENDED, but not required.
+  description: Datetime format and their anonymization are described in [Units](/02-common-principles.html#units).
```

Comment:
```suggestion
  description: |
    The acquisition time for the first data point in each element of the file. 
    The datetime format and its anonymization are described in [Units](/02-common-principles.html#units).
```

Reply:
Per today's maintainers call, splitting the definitions is probably the way to go, so you can ignore this suggestion.